packages/remirror__styles/all.css

```diff
@@ -2963,6 +3031,10 @@ button:active .remirror-menu-pane-shortcut,
   overflow: auto;
 }
 
+.remirror-pojoaque :has(.remirror-language-select-positioner) ~ pre[class*='language-'] {
+  padding: 2em 1em;
```

Comment:
Not sure why some themes use different padding values for the pre element. For example the "pojoaque" theme uses 12px instead of the 1em. 

I think the padding I added to add some spacing for the language select can be adjusted in another pr if needed.

Reply:
What do you think we don't add padding in `@remirror/styles`, but add some example of CSS code in the readme (and the storybook) to show how to add such padding? I guess whoever wants to use `@remirror/extension-react-language-select` would probably want to add some custom CSS anyway.

Reply:
I guess maybe some theme re-define `1em`? I'm not sure. I think it's fine since the user might want to add some custom CSS by themselves anyway, so we don't have to make everything perfect.